### ISSUE TITLE
Win 10 Ie11 : Unable to set/get window size upgrading from IEDriver 3.4 to IEDriver 3.6

### ISSUE THREAD

**Issue by abioneperhobby** [SeleniumHQ/selenium#4937]
On
Windows 10 64 bit full stack updated
With
Selenium 3.6
IEDriver 3.6

this

```
from selenium import webdriver
browser = webdriver.Ie()
browser.set_window_size(1300, 1000)
```
will not change window size and this

`print(browser.get_window_size())`

returns

> {'message': 'Command not found: GET /session/cca537b7-5bc0-49ee-972e-400d9cf1ddb5/window/size', 'value': None, 'error': 'unknown method'}

On Chrome set_window_size works and get_window_size returns the right dimension.

This is a regression , because with IEDriver 3.4, it works without error also on IE11.

**Comment by pekkaklarck** [comment]
Somewhat related to this, is there a reason Selenium doesn't raise an exception when a driver returns such an error? Browsing through the code it seems that an error should be raised by [ErrorHandler.check_response()](https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/remote/errorhandler.py#L80) which ought to be called by [WebDriver.execute()](https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/remote/webdriver.py#L288) that [WebDriver.get_window_size()](https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/remote/webdriver.py#L959) ought to call. Based on the above description, it seems no error is raised and invalid data is simply returned to the user.

**Comment by p0deje** [comment]
It looks like Python bindings implement incorrect commands for getting/setting window size:

```python
            Command.W3C_GET_WINDOW_SIZE:
                ('GET', '/session/$sessionId/window/size'),
            Command.W3C_SET_WINDOW_SIZE:
                ('POST', '/session/$sessionId/window/size'),
```

No such commands are defined by [specification](https://w3c.github.io/webdriver/webdriver-spec.html#resizing-and-positioning-windows), so bindings should instead fallback to rect commands. [Just like Ruby bindings do](https://github.com/SeleniumHQ/selenium/blob/9eb5f0bd33284b7a9779d4f42fb79d36cf728b07/rb/lib/selenium/webdriver/remote/w3c/bridge.rb#L164-L178).

**Comment by p0deje** [comment]
@pekkaklarck Interesting, can you please raise a separate issue for that?

**Comment by pekkaklarck** [comment]
@p0deje Done. See #4948.

**Comment by lmtierney** [comment]
Fixed in ffce538, will be in the next release. I'll look at the other issue you added

**Comment by abioneperhobby** [comment]
with version 3.8 getter and setter works.

```
C:\Python27\python.exe C:/src/python_selenium/set_window_size.py
{'width': 1936, 'height': 1056}

```
![image](https://user-images.githubusercontent.com/2149815/33943314-0d4adb58-e019-11e7-8128-632b37b12370.png)
